core/pom.xml

```diff
@@ -646,7 +646,7 @@
 		<dependency>
 			<groupId>org.apache.tika</groupId>
 			<artifactId>tika-core</artifactId>
-			<version>1.28.5</version>
+			<version>2.9.2</version>
 			<exclusions>
 				<exclusion><!-- excluded because we want to ignore this (1.2.2) version! this dependency handles javax.activation.* imports, and is NOT compatible with the IBM JRE -->
```

Comment:
Are you sure this exclusion is still needed? In the above `aspose/pom.xml`, you've removed it.

Reply:
Yes, but I was just thinking that this library isn't a dependency of the tika-core anymore.

Reply:
Good point, I'll remove the comment and exclusion